modules/jt-comparing-jenkins-tekton-concepts.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assembly:
+//
+// jenkins-tekton/migrating-from-jenkins-to-tekton.adoc
+
+[id="jt-comparing-jenkins-tekton-concepts_{context}"]
+= Comparing Jenkins and Tekton concepts
+
+toc::[]
+
+
+This section summarizes the basic terms used in Jenkins and Tekton, and compares the equivalent terms.
+
+== Jenkins terminology
+Jenkins offers declarative and scripted pipelines that are extensible using shared libraries and plugins. Some basic terms in Jenkins are as follows:
+
+* *Pipeline*: Automates the entire process of building, testing, and deploying applications, using the link:https://groovy-lang.org/[Groovy] syntax.
+* *Node*: A machine capable of orchestrating and/or executing a scripted pipeline.
+* *Stage*: A conceptually distinct subset of tasks performed in a pipeline. This block is often used by plugins or user interfaces to display status or progress of tasks.
+* **Step**: A single task that specifies the exact action to be taken, either by using a command or a script.
+
+== Tekton terminology
+Tekton uses the link:https://yaml.org/[YAML] syntax for declarative pipelines and consists of tasks. Some basic terms in Tekton are as follows:
+
+* **Pipeline**: A set of tasks that may execute serially, in parallel, or both.
+* **Task**: A sequence of steps that are executed as commands, binaries, or scripts.
+* **PipelineRun**: Execution of a pipeline with one or more tasks.
+* **TaskRun**: Execution of a task with one or more steps.
++
+[NOTE]
+====
+You can initiate a PipelineRun or a TaskRun with a set of inputs such as parameters and workspaces, and the execution results in a set of outputs and artifacts.
+====
+* **Workspace**: In Tekton, workspaces are conceptual blocks that serves the following purposes:
+
+** Storage of inputs, outputs, and build artifacts
```

Comment:
All your other bullet points include periods at the end of the sentence. So I would include periods at the end of your subbullets too. Especially since you have a period at the end of the third subbullet, even though grammatically it's not a complete sentence.